src/main/java/Duke.java

```diff
@@ -1,10 +1,130 @@
+                    }else{
+                        System.out.println(SPLIT_LINE);
+                        System.out.println(" Got it. I've added this task:");
+                        tasks.add(new Deadline(line));
+                        System.out.println("  "+new Deadline(line));
+                        System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+                        System.out.println(SPLIT_LINE);
+                    }
+                }
+                else if(line.startsWith("event")){
+                    if(line.equals("event")){
+                        throw new Exception();
+                    }else{
+                        System.out.println(SPLIT_LINE);
+                        System.out.println(" Got it. I've added this task:");
+                        tasks.add(new Event(line));
+                        System.out.println("  "+new Event(line));
+                        System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+                        System.out.println(SPLIT_LINE);
+                    }
+                }
+                else if(line.startsWith("delete")){
+                    System.out.println(SPLIT_LINE);
+                    int taskNumberDeleted = Integer.parseInt(line.replaceAll("\\D+",""))-1;
+                    tasks.set(taskNumberDeleted,tasks.get(taskNumberDeleted).completeTask());
+                    System.out.println(" Noted. I've removed this task: ");
+                    System.out.println("  "+tasks.get(taskNumberDeleted));
+                    tasks.remove(taskNumberDeleted);
+                    System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+                    System.out.println(SPLIT_LINE);
+                }
+                else {
+                    System.out.println(SPLIT_LINE);
+                    System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
+                    System.out.println(SPLIT_LINE);
+                }
+            }catch (Exception e){
+                if(line.equals("todo"))
+                {   System.out.println(SPLIT_LINE);
+                    System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
+                    System.out.println(SPLIT_LINE);
+                }
+                if(line.equals("deadline"))
+                {   System.out.println(SPLIT_LINE);
+                    System.out.println("☹ OOPS!!! The description of a deadline cannot be empty.");
+                    System.out.println(SPLIT_LINE);
+                }
+                if(line.equals("event"))
+                {   System.out.println(SPLIT_LINE);
+                    System.out.println("☹ OOPS!!! The description of a event cannot be empty.");
+                    System.out.println(SPLIT_LINE);
+                }
+            }
+            if(sc.hasNextLine()){
+                line=sc.nextLine();}
```

Comment:
To enhance readability, you may consider putting a space before and after the `=`. This goes for all the other statements as well, such as line 113.